app.js

```diff
@@ -0,0 +1,15 @@
+var express = require('express');
+
+var app = express();
+app.set('port', (process.env.PORT || 5000));
+app.use(express.static(__dirname + '/dist'));
+
+app.set('views', __dirname + '/dist');
+app.set('view engine', 'ejs');
+app.get('/', function (req, res) {
```

Comment:
Might be good to change this to a all-paths wildcard so we can use HistoryToken in react-router